db/src/main/kotlin/edu/artic/db/models/ModelExtensions.kt

```diff
@@ -0,0 +1,13 @@
+package edu.artic.db.models
```

Comment:
Rename file (too generic) or put it in the same file as `ArticObject` as an extension method residing near the class.

Reply:
Removed this file and moved the extension method to ArticObject as the extension property.